### PR TITLE
Created bug template 

### PR DESCRIPTION
**Pull request by RishiKumarRay**
Created Bug template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Bug
+description: Report an issue that would help to improve this project.
+labels: [' goal: fix']
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/Bug.yml:3]:
```suggestion
labels: ['bug']
```

File: .github/ISSUE_TEMPLATE/Bug.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue.
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/Bug.yml:9]:
```suggestion
      description: A brief description of the problem.
```

File: .github/ISSUE_TEMPLATE/Bug.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      label: Additional information
+      description: Is there anything else we should know about this bug?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can Submit this issue now.
+       
+
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/Bug.yml:31]: Please add newline at the end of the file.

```suggestion
    
```

File: .github/ISSUE_TEMPLATE/Bug.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bug?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can Submit this issue now.
+
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/Bug.yml:30]: This is probably not needed, right? The submission button will appear anyway.

```suggestion
```

File: .github/ISSUE_TEMPLATE/Bug.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    attributes:
+      label: Description
+      description: A brief description of the question or issue.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add any screenshots if applicable
```
Comment (kirillbobyrev) [inline .github/ISSUE_TEMPLATE/Bug.yml:16]: For a chess engine, I guess the logs are more informative than screenshots, may be worth asking for that instead.

```suggestion
    attributes:
      label: Logs
      description: Please add the logs with `RUST_BACKTRACE=1` and `RUST_LOG=info`.
```

**Review by kirillbobyrev — COMMENTED** [review]
Thanks a lot for the help! I left few comments: it looks good overall, but it might be nice to add some placeholders from the default "Bug" template that has this along the lines:

```markdown
**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Additional context**
Add any other context about the problem here.
```

Of course, not all of it is applicable but it looks quite neat. Issue default title is another interesting example of a cool feature (e.g. starting with the [BUG], although there are tags for filtering)!

Creating the templates from the default ones seems quite easy: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-templates

**Comment by RishiKumarRay** [comment]
@kirillbobyrev  I have done the changes

**Review by kirillbobyrev — APPROVED** [review]
Looks Good To Me, thank you!

**Pull request merged**